include/eve/module/real/core/function/regular/simd/x86/add.hpp

```diff
@@ -0,0 +1,63 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/detail/implementation.hpp>
+#include <eve/function/saturated.hpp>
+
+#include <type_traits>
+
+namespace eve::detail
+{
+  // -----------------------------------------------------------------------------------------------
+  // 128 bits implementation
+  template<real_scalar_value T, typename N>
+  EVE_FORCEINLINE wide<T, N> add_(EVE_SUPPORTS(sse2_), wide<T, N> v0, wide<T, N> v1) noexcept
+    requires x86_abi<abi_t<T, N>>
+  {
+    return v0 += v1;
+  }
```

Comment:
Why is this necessary? I mean the base case should still work, right?

Reply:
perhaps useless I did it as it was in sub

Reply:
Maybe remove it everywhere if you don't mind?
Just: less code - easier to maintain?